webapp/javascript/components/TimelineChart/ExploreTooltip.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo, FC } from 'react';
+import classNames from 'classnames/bind';
+import Color from 'color';
+import styles from './ExploreTooltip.module.scss';
+
+const cx = classNames.bind(styles);
+
+const DEFAULT_COMPONENT_WIDTH = 450;
+const EXPLORE_TOOLTIP_WRAPPER_ID = 'explore_tooltip_wrapper';
+
+export interface ExploreTooltipProps {
+  pageX: number;
+  pageY: number;
+  align: 'left' | 'right';
+  timeLabel?: string;
+  values?: Array<{
+    closest: number[];
+    color: number[];
+    tagName: string;
+  }>;
```

Comment:
I don't fully understand this, why do we receive the `color/tagName`? what is a `tagName` in a timeline?
for example, in this timeline what's a `tagName`?

![image](https://user-images.githubusercontent.com/6951209/188171543-26acd2b6-333f-4e75-ab29-31ccf6f46972.png)


https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/flot/index.d.ts#L157-L164

Reply:
since we splitted tooltip into 2 parts, `color/tagName` belong only part related to Explore page, on another pages we get time and nearest data so we can customize it as we want